services/pulls/pulls.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package pulls
```

Comment:
It should probably be named `pull` to have same naming as it already is for `modules/pull`